src/utils/config.py

```diff
@@ -14,6 +15,24 @@
 _CONFIG_SINGLETON = None
 
 
+def get_sample_service_url(sw_url):
+    """"""
+    payload = {
+        "method": "ServiceWizard.get_service_status",
+        "id": '',
+        "params": [{"module_name": "SampleService", "version": "dev"}],  # TODO: change to beta/release
+        "version": "1.1"
+    }
+    headers = {'Content-Type': 'application/json'}
+
+    sw_resp = requests.post(url=sw_url, headers=headers, data=json.dumps(payload))
```

Comment:
I would also either do `sw_resp.raise_for_status()` or do a conditional on `if not sw_resp.ok` to handle a non 2xx response